gpjax/parameters.py

```diff
@@ -173,21 +172,35 @@ def inverse(bijector):
     return constrainers, unconstrainers
 
 
-def transform(params: tp.Dict, transform_map: tp.Dict) -> tp.Dict:
+def transform(params: Dict, transform_map: Dict) -> Dict:
     """Transform the parameters according to the constraining or unconstraining function dictionary.
 
     Args:
-        params (tp.Dict): The parameters that are to be transformed.
-        transform_map (tp.Dict): The corresponding dictionary of transforms that should be applied to the parameter set.
+        params (Dict): The parameters that are to be transformed.
+        transform_map (Dict): The corresponding dictionary of transforms that should be applied to the parameter set.
 
     Returns:
-        tp.Dict: A transformed parameter set.s The dictionary is equal in structure to the input params dictionary.
+        Dict: A transformed parameter set.s The dictionary is equal in structure to the input params dictionary.
     """
     return jax.tree_util.tree_map(
         lambda param, trans: trans(param), params, transform_map
     )
 
 
+def build_identity(params: Dict) -> Dict:
+    """ "
+    Args:
+        params (Dict): The parameter set for which trainable statuses should be derived from.
+
+    Returns:
+        Dict: A dictionary of identity forward/backward bijectors. The dictionary is equal in structure to the input params dictionary.
+    """
+    # Copy dictionary structure
+    prior_container = deepcopy(params)
+
+    return jax.tree_util.tree_map(lambda _: Identity.forward, prior_container)
+
+
```

Comment:
I don't get why this is needed?

Reply:
In an upcoming commit, I have changed `gpx.transform` to accept `transformation_map = None` to give an identity transformation.

Reply:
Resolved in https://github.com/thomaspinder/GPJax/commit/cb45fbfc3ffa449cb216c11250edb0fe4f20051b.